src/views/Change.vue

```diff
@@ -2,7 +2,7 @@
   <section class="pb-10" id="change-view">
     <!-- Business Information page-->
     <v-slide-x-transition hide-on-leave>
-      <div v-if="!isSummaryMode">
+      <div v-if="!isSummaryMode || !showFeeSummary">
```

Comment:
This is to show the Business Information page if nothing has been changed. I think this is only valid while debugging and hot-reloading the app (which loses previous changes) but I think it doesn't hurt either.